test/unit/orm-adapters-memory-test.js

```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
```

Comment:
I'm fine with this, or going nested. I imagine nested will be the eventual solution, but I don't mind starting simple 😄